### PR TITLE
Update getting-started-basic.mdx to use llama_cpp instead of openai

### PR DESCRIPTION
**Pull request by rishiraj**

### CODE REVIEW

File: docs/docs/getting-started-basic.mdx [hunk 1]
```diff
@@ -161,16 +171,49 @@ indexify-extractor join-server
...
+      overlap: 100
+  - extractor: "tensorlake/minilm-l6"
+    name: "wikiembedding"
+    content_source: "chunker"
+```
+```yaml graph.yaml for OpenAI
 name: "wiki_extraction_pipeline"
 extraction_policies:
   - extractor: "tensorlake/openai"
+    name: "entity-extractor"
```
Comment (diptanu) [inline docs/docs/getting-started-basic.mdx:200]: You could do Content Tabs on Mintlify and use code blocks inside them. So you can have three blocks - Local Model, OpenAI and Mistral.

File: docs/docs/getting-started-basic.mdx [hunk 2]
```diff
@@ -267,18 +310,72 @@ You can query Indexify to -
...
 
 For now let us create a file named `query.py` with the following content:
 
-```python query.py
+```python query.py for Llama.CPP
+from indexify import IndexifyClient
+from llama_cpp import Llama
+
+client = IndexifyClient()
+llm = Llama.from_pretrained(repo_id='microsoft/Phi-3-mini-4k-instruct-gguf', filename='*q4.gguf', verbose=False)
```
Comment (diptanu) [inline docs/docs/getting-started-basic.mdx:332]: Why not Q8? Wouldn't that work better?

**Review by diptanu — CHANGES_REQUESTED** [review]

File: docs/docs/getting-started-basic.mdx (hunk 2 above)
Comment (rishiraj) [reply docs/docs/getting-started-basic.mdx:332]: Q8 would become 4.4 GB file download (double of current 2.2 GB) for a very minute difference in results. Also Microsoft provides just q4 and fp16 (around 8 GB) gguf files.

File: docs/docs/getting-started-basic.mdx (hunk 1 above)
Comment (rishiraj) [reply docs/docs/getting-started-basic.mdx:203]: done

**Pull request merged**